controllers/leds.py

```diff
@@ -15,22 +15,18 @@ class LedController:
     vision: Vision
 
     def execute(self) -> None:
-        if not self.vision.is_ready():
-            self.status_lights.pulse(LedColours.PINK)
```

Comment:
This means that the drive team cannot tell whether the vision is up when they put the robot on the field.